### ISSUE TITLE
Verrryyyy slow start and clock out of sync after sleep

### ISSUE THREAD

**Issue by Krzysiaczek-at-theFoundry** [docker/for-mac#1391]
Docker for Mac: version: 17.03.0-ce-mac1 (e5a07a1a6)
macOS: version 10.12.3 (build: 16D32)

Two annoying problems experienced:
- container clock is out of sync - even a couple of days after machine goes to sleep
- incredible long start time - around a minute

I am using docker-compose up to run multiple services

**Comment by MagnusS** [comment]
Thanks for the report. I've removed the logs from your message - could you please upload them again using the built-in diagnostics feature and report the ID here? To upload the logs select "Diagnose & Feedback" from the menu, then click "Diagnose and Upload".

The time sync issue may be the issue reported in #17. Is the long start time for Docker for Mac itself?

**Comment by Krzysiaczek-at-theFoundry** [comment]
New diagnostic ID just uploaded: A2B312AF-AC92-409B-A35B-CEE103198DEB
the part of the name of the file you removed had previous diagnostic ID which run during this slow restart process.

And yes by the long start I meant time for get any response from container. This is fresh new powerful I7 machine to which I switched from 2009 DualCore Mac running Toolbox and on which I didn't experienced any problems like this.

**Comment by Krzysiaczek-at-theFoundry** [comment]
And yes #17 seems like the same issue however I found similar reports https://forums.docker.com/t/time-in-container-is-out-of-sync/16566 which are very old so the problem seems to be known for quite a while.

**Comment by jamshid** [comment]
I wish the time sync issue were fixed but until then I have this in my ~/.profile and I run it when needed.
```
alias fixdockerformac="docker run --rm --privileged alpine hwclock -s"
```

On the slowness, I wonder if it's the same problem I just had in https://github.com/docker/for-mac/issues/1395. As discussed in  docker/compose#3419 (comment) it's fixed by adding this to your /etc/hosts. You might also need localunixsocket.local, though that didn't help me.
```
127.0.0.1 localunixsocket
```

**Comment by Krzysiaczek-at-theFoundry** [comment]
I don't know what to say as today everything seems to fine. Clock is synced and I've got instance response from docker-compose up, weird.

**Comment by MagnusS** [comment]
@Krzysiaczek-at-theFoundry thanks for the update. Good to hear that compose is working again. We are working on the time sync issue and hopefully there will be a better fix available soon.  I'll close this issue as the original problem is resolved - feel free to reopen if the problem reappears.

**Comment by Krzysiaczek-at-theFoundry** [comment]
Today I realised that both problems are back and it's not due to putting computer to sleep as I thought initially but due to turning off docker by command
`docker-compose stop`
I run that command before the weekend.
Today when I wanted to start it by 
`docker-compose up`
I realise that it takes too long to get any answer so I turned on the stopper to catch the time. It took more than 1:30 minutes to get first respond from first service so I think the overall time was about 2 minutes. On other days when I stop it just for a while to restart after some changes in docker-compose.yml the responses from first and all other container come in no time and whole restart process takes just a couple of seconds.
The clock inside the docker shows now some time from 10th of March
